client/src/components/App.jsx

```diff
@@ -43,43 +49,53 @@ class App extends React.Component {
     document.removeEventListener('mousedown', this.handleClickOutside);
   } */
 
-  setWrapperRef(node) {
-    console.log('inside wrapper');
+  /* setWrapperRef(node) {
     this.wrapperRef = node;
-    console.log(node);
-  }
+  } */
 
   handleOnChange(e) {
     const stateObj = this.state;
     if (stateObj.isVisible) {
       this.setState({ isVisible: true });
     }
 
-    const userId = 1;
+    const { videoId } = this.state;
     const categoryId = e.target.value;
-    fetch(`filter/${userId}/${categoryId}`)
+    fetch(`filter/${videoId}/${categoryId}`)
       .then((response) => response.json())
       .then((data) => {
         this.setState({
-          username: data.user.username,
           videos: data.videos,
           currentCategory: data.videos[0].category.name,
         });
       });
   }
 
   handleOnRightClick() {
-    document.getElementById('container').scrollLeft += 50;
+    const oldScroll = document.getElementById('container').scrollLeft;
+    document.getElementById('container').scrollLeft += scrollDistance;
```

Comment:
Hmmm...updating the DOM directly doesn't seem very react-y, but I'm at a loss of what to suggest.

Reply:
I agree. I want to find a better solution. Thanks @Kraig5.